docs/source/reference/configuration.rst

```diff
@@ -66,6 +66,11 @@ Configuration Keys
 
    You can change the default value to ``True`` by setting ``CHAINER_KEEP_GRAPH_ON_REPORT`` environment variable to ``1``.
 
+* ``warn_nondeterministic`` (default: ``False``)
+   Flag to give warning when a non-deterministic function is used. This function is experimental.
+
+   If it is true, then functions that use non-deterministic functions, such as atomicAdd, will give a warning when executed. Note the determinisitic outputs may still result, despite using a nondeterministic function, depending on usage.
```

Comment:
We should mention false negatives first, which are more important.
Let's discuss the expression offline later.